pages/index.tsx

```diff
@@ -28,7 +28,17 @@ export default function HomePage({ metricsData, siteConfigData }: HomeProps) {
 
 export const getStaticProps: GetStaticProps<HomeProps> = async () => {
   const siteConfigData = CONFIG
-  const { volumeUsd, volumeEth } = await cowSdk.cowSubgraphApi.getTotals()
+  let volumeUsd = 0
+  let volumeEth = 0
+
+  // Don't fail when couldn't get Subgraph data
```

Comment:
Shouldn't this be an isolated PR in itself?
Then we can release it independently from the T&C

Reply:
https://github.com/cowprotocol/cow-fi/pull/170